docs/reference/mount.md

```diff
@@ -88,10 +93,37 @@ The `umount` command is used to unmount a currently mounted lakeFS repository.
 everest umount <data_directory>
 ````
 
+### Diff Command (write-mode only)
+The `diff` command Show the diff between the source branch and the current mount directory. 
+If `<data_directory>` not specified, the command searches for the mount directory in the current working directory and upwards based on `.everest` directory existence.
+Please note that the diffs are from the source branch state at the time of mounting and not the current state of the source branch, i.e., changes to the source branch from other operations will not be reflected in the diff result.
+
+```bash
+everest diff <data_directory>
+
+#Example output:
+# - removed cats/persian/cute.jpg
+# ~ modified dogs/golden_retrievers/cute.jpg
+# + added birds/parrot/cute.jpg
+```
+
+### Commit Command (write-mode only)
+The `commit` command commits the changes made in the mounted directory to the original lakeFS branch.
+If `<data_directory>` not specified, the command searches for the mount directory in the current working directory and upwards based on `.everest` directory existence.
+The new commit will be merged to the original branch with the `source-wins` strategy in case of conflicts.
+After committing, the mounted directory will be updated with the HEAD of the source branch; that means that changes made to the source branch out of the mount scope will also be reflected in the mounted directory.
+****
+⚠️ Writes to a mount directory during commit may be lost.
```

Comment:
Incorrect render all the commit content appears as a notification
![image](https://github.com/user-attachments/assets/55fa70aa-56dd-44b0-ad2f-12c36baec61e)